apps/services/search-indexer/dev-services/README.md

```diff
@@ -1,51 +1,50 @@
 # Elasticsearch Search Dev Server
 
-### Dependencies
+## Dependencies
 
-You must have `docker` and `docker compose` installed and running on you machine.  
-**Docker must be given 4gb+ of ram**
+Ensure `docker` and `docker compose` are installed, with Docker allocated 4GB+ RAM.
 
-### Start the server
+## Start the Server
 
 ```bash
 yarn dev-services services-search-indexer
 ```
 
-In about 30 seconds
-You should have Elasticsearch running on `localhost:9200`  
-And Kibana running on port `localhost:5601`
+Wait ~30 seconds. Access Elasticsearch at `localhost:9200` and Kibana at `localhost:5601`.
 
-### Import data
+## Import Data
 
-**You must have access to Contentful for the indexer to work correctly**  
-To import map templates, create indice and import data run
+**Contentful access is required.**
 
-Requred env variables:
+Set environment variables:
 
-- `CONTENTFUL_HOST=preview.contentful.com` - Get from Contentful
-- `CONTENTFUL_ACCESS_TOKEN=` - Get from Contentful or by running `yarn get-secrets api`
+- `CONTENTFUL_HOST=preview.contentful.com`
```

Comment:
This could also be "https://cdn.contentful.com" in case the dev wants to fetch "published data" from Contentful.
But to do that he'll need that kind of access token